pubspec.yaml

```diff
@@ -8,11 +8,11 @@ homepage: https://Apparence.io
 repository: https://github.com/Apparence-io/camera_awesome
 
 environment:
-  sdk: ">=3.1.0 <4.0.0"
+  sdk: '>=3.2.6 <4.0.0'
   flutter: ">=1.10.0"
 
 dependencies:
-  carousel_slider: ^4.2.1
```

Comment:
The `carousel_slider` package got a 5.0.0 release just the other day and is much more widely used than the fork. Probably sticking with the original is the better choice here.

Reply:
@kuhnroyal @biklas7
Done in https://github.com/Apparence-io/CamerAwesome/pull/504 (I branched from your branch @tonistayhungrystayfoolish )

Reply:
@kuhnroyal we now have three open prs on this repo that claim to fix this issue. When are we going to get an update on it being fixed? Your package is now incompatible with the **latest stable** version of flutter.

Reply:
@GTechMicah I am not involved here. I only looked over the open PR since I also need the change.

Reply:
Oh Sorry, thanks for clarifying